Example/Podfile

```diff
@@ -1,21 +1,21 @@
-source 'https://github.com/CocoaPods/Specs.git'
+ 'https://github.com/CocoaPods/Specs.git'
 
 target 'OCSlimProject_AcceptanceTests' do
   platform :ios, 9.0
-  pod "OCSlimProject", :git => "https://github.com/paulstringer/OCSlimProject.git", :tag => "1.3"
+  pod "OCSlimProject", :git => "https://github.com/paulstringer/OCSlimProject.git", :branch => "npm-begone"
```

Comment:
Probably want to update these references to point to master?

Reply:
This is required until a new release is cut from master otherwise this PR build will fail till merged as the checkout results in pulling from an older version. The dependancy update happens on master before taking a cut of a new release. 👍